save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsView.kt

```diff
@@ -277,7 +275,7 @@ abstract class UserSettingsView : AbstractView<UserSettingsProps, UserSettingsVi
                                                 className = ClassName("mt-2")
                                                 Link {
                                                     className = ClassName("item")
-                                                    to = "/${props.userName}/${FrontendRoutes.SETTINGS_TOKEN}"
+                                                    to = "/${props.userInfo}/${FrontendRoutes.SETTINGS_TOKEN}"
```

Comment:
Didn't know `UserInfo` has `toString` overwritten

Reply:
these views are not needed anymore, they will die, I wanted to split their deletion, but Idea is merging conflicts in very weird way

Reply:
ok then